lib/src/folding/parsers/fallback.dart

```diff
@@ -99,30 +113,94 @@ class FallbackFoldableBlockParser extends TextFoldableBlockParser {
                 }
                 break;
               }
+              if (c.contains(char)) {
+                _isPossibleCommentSequence = true;
+              }
+            }
+
+            if (isMultilineCommentingEnabled) {
+              for (final c in multilineCommentSequences!) {
+                if (tail.endsWith(c.item1)) {
+                  if (!serviceCommentLines.contains(lineIndex)) {
+                    if (!foundImportTerminator) {
+                      startBlock(lineIndex, FoldableBlockType.multilineComment);
+                      _shouldEndMultilineComment = true;
+                    } else {
+                      // shouldn't start multiline comment block
+
+                      // class MyClass { /*
+                      // */
+                      // }
+                      _shouldEndMultilineComment = false;
+                    }
+                    _startedMultilineCommentWith = c.item1;
+                    _isInMultilineComment = true;
+                    break;
+                  }
+                }
+                if (c.item1.contains(char)) {
+                  _isPossibleCommentSequence = true;
+                }
```

Comment:
Can we change the HighlightFoldableBlockParser so that /* */ comments are not treated as single line comments? Will this algorithm be simpler then?